dashboard/app/controllers/rubrics_controller.rb

```diff
@@ -46,10 +46,30 @@ def update
   def submit_evaluations
     return head :forbidden unless current_user&.teacher?
     permitted_params = params.permit(:id, :student_id)
+    rubric = Rubric.find(permitted_params[:id])
     learning_goal_ids = LearningGoal.where(rubric_id: permitted_params[:id]).pluck(:id)
     evaluations = LearningGoalTeacherEvaluation.where(user_id: permitted_params[:student_id], learning_goal_id: learning_goal_ids, teacher_id: current_user.id)
```

Comment:
out of scope of this PR I think but we should update this to only update unsubmitted evaluations. task here: https://codedotorg.atlassian.net/browse/AITT-292